src/util-device.h

```diff
@@ -49,6 +49,7 @@ typedef struct {
 typedef struct LiveDevice_ {
     char *dev;  /**< the device (e.g. "eth0") */
     char dev_short[MAX_DEVNAME + 1];
+    uint8_t mtu; /* MTU of the device */
```

Comment:
incorrect type. Common mtu is 1500, we also have 9000. Should be at least uint16_t, but perhaps just make it `int`

Reply:
even 1500 wouldn't have fitted in u8

Reply:
indeed 🤦🏽